src/components/container/SurveyTaskList/SurveyTaskList.tsx

```diff
@@ -13,7 +13,8 @@ export interface SurveyTaskListProps {
 	surveys?: string[],
 	onDetailLinkClick?: Function,
 	previewState?: SurveyTaskListListPreviewState
-	variant?: "noCard" | "singleCard" | "multiCard"
+	variant?: "noCard" | "singleCard" | "multiCard",
+	hideIfNoTasks?: boolean
```

Comment:
It doesn't look like this get used.

Reply:
Oops I think this was accidentally committed when I was starting to work on a different thing